charts/datadog/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: datadog
-version: 3.56.0
+version: 3.56.1
```

Comment:
We normally increase the minor version when updating the images, so it should be 3.57.0.

Reply:
Got it 👍 Thanks! I've done the modification 😄